Core/include/Acts/Seeding/SeedFinderUtils.ipp

```diff
@@ -188,19 +188,25 @@ inline bool xyzCoordinateCheck(
   const Acts::Vector3& stripCenterDistance =
       spacePointData.getStripCenterDistance(index);
 
+  // prepare variables
+  double xTopStripVector = topHalfStripLength * topStripDirection[0];
+  double yTopStripVector = topHalfStripLength * topStripDirection[1];
+  double zTopStripVector = topHalfStripLength * topStripDirection[2];
+  double xBottomStripVector = bottomHalfStripLength * bottomStripDirection[0];
+  double yBottomStripVector = bottomHalfStripLength * bottomStripDirection[1];
+  double zBottomStripVector = bottomHalfStripLength * bottomStripDirection[2];
```

Comment:
good point! thanks

Reply:
Can you disentagle this change from the rest of this PR? This will improve performance for SSS and is not controversial. I'd suggest opening a PR only with this change so that it can be accepted quickly